pkgs/data/fonts/iosevka/default.nix

```diff
@@ -1,83 +1,61 @@
-{
-  stdenv, lib, pkgs,
-  fetchFromGitHub, fetchurl,
-  nodejs, ttfautohint-nox, otfcc,
+{ stdenv, lib, pkgs, fetchFromGitHub
+, nodejs, nodePackages, remarshal, ttfautohint-nox, otfcc
 
-  # Custom font set options.
-  # See https://github.com/be5invis/Iosevka#build-your-own-style
-  design ? [], upright ? [], italic ? [], oblique ? [],
-  family ? null, weights ? [],
-  # Custom font set name. Required if any custom settings above.
-  set ? null,
-  # Extra parameters. Can be used for ligature mapping.
-  extraParameters ? null
+# Custom font set options.
+# See https://github.com/be5invis/Iosevka#build-your-own-style
+, privateBuildPlan ? null
+# Extra parameters. Can be used for ligature mapping.
+, extraParameters ? null
+# Custom font set name. Required if any custom settings above.
+, set ? null
 }:
 
-assert (design != []) -> set != null;
-assert (upright != []) -> set != null;
-assert (italic != []) -> set != null;
-assert (oblique != []) -> set != null;
-assert (family != null) -> set != null;
-assert (weights != []) -> set != null;
+assert (privateBuildPlan != null) -> set != null;
 
-let
-  system = builtins.currentSystem;
-  nodePackages = import ./node-packages.nix { inherit pkgs system nodejs; };
-in
+stdenv.mkDerivation rec {
+  pname =
+    if set != null
+    then "iosevka-${set}"
+    else "iosevka";
 
-let pname = if set != null then "iosevka-${set}" else "iosevka"; in
-
-let
   version = "2.3.0";
-  name = "${pname}-${version}";
+
   src = fetchFromGitHub {
     owner = "be5invis";
-    repo ="Iosevka";
-    rev = "v${version}";
+    repo = "Iosevka";
+    rev = version;
```

Comment:
Isn't `rev` supposed to be `v${version}` ?

Reply:
That's weird, I built the derivation with just `version` but you're right that it shouldn't exist. I wonder if caching made something odd happen with my build?

Reply:
Yes, it's a fixed output derivation, so it won't be rebuilt if a derivation with the correct hash already exists in your Nix store.